src/core/Akka/Serialization/Serializer.cs

```diff
@@ -121,7 +121,17 @@ public static int GetSerializerIdentifierFromConfig(Type type, ExtendedActorSyst
         {
```

Comment:
Has this replaced the use of `serializer.Identifier` through out the Scala code? or are they assigning the value to the Identifier property using this?

(It seems to be a strange way to go about things as you might end up with one type of serializer on one node claiming to be identifier 1, and a completely different serializer with the same identifier on another node.)